test_project/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "electron-prebuild-test",
+  "main": "index.js",
+  "version": "0.1.0",
+  "repository": "https://github.com/electron/electron-rebuild",
+  "license": "MIT",
+  "scripts": {
+    "start": "electron .",
+    "rebuild": "electron-rebuild -f -p --verbose"
+  },
+  "devDependencies": {
+    "electron": "^1.4.3",
+    "electron-rebuild": "github:electron/electron-rebuild"
```

Comment:
This should not work and should be be reccomended in the source code.  Because our output is `babelified` just cloning from github will not result in a workable module.